fvtest/compilertriltest/IfxcmpgeReductionTest.cpp

```diff
@@ -24,11 +24,15 @@
 #include "JitTest.hpp"
 #include "jitbuilder_compiler.hpp"
 
-template <typename ValType>
-using IfxcmpgeReductionParamType = std::tuple<ValType, ValType, int32_t (*)(ValType, ValType)>;
+// TODO: This kind of indirection requires C++11 features and is nontrivial
+//       to emulate without complicating the code. We should eventually turn
+//       this back on (and modify the associated function signatures) once we
+//       have moved to a C++11 supported compiler.
```

Comment:
I renew my objection to using TODOs in the code, and in particular for some event (enabling code) that must happen when another event (upgrading the build compiler) occurs. Please create an issue for this so that this action and rationale won't be lost in the code, and reference the issue here instead.  This could very well be the same issue for all the build compiler TODOs in this PR.